nano/test_common/system.cpp

```diff
@@ -154,27 +154,27 @@ void nano::system::ledger_initialization_set (std::vector<nano::keypair> const &
 	nano::block_hash previous = nano::genesis_hash;
 	auto amount = (nano::genesis_amount - reserve.number ()) / reps.size ();
 	auto balance = nano::genesis_amount;
-	for (auto const & i: reps)
+	for (auto const & i : reps)
 	{
 		balance -= amount;
 		nano::state_block_builder builder;
 		builder.account (nano::dev_genesis_key.pub)
-		       .previous (previous)
-		       .representative(nano::dev_genesis_key.pub)
-		       .link (i.pub)
-		       .balance (balance)
-		       .sign (nano::dev_genesis_key.prv, nano::dev_genesis_key.pub)
-		       .work (*work.generate (previous));
+		.previous (previous)
```

Comment:
are these changes supposed to go?

Reply:
These were automatically generated when I ran the clang format script, so I guess yes, actually I'm not even sure why the file wasn't correctly formatted.